priam/src/main/java/com/netflix/priam/defaultimpl/StandardTuner.java

```diff
@@ -165,6 +169,16 @@ private void configureCommitLogBackups() throws IOException {
         }
     }
 
+    private void configureBatchSizes(CassandraConfiguration cassandraConfiguration, Map<String, Object> yaml, @Nullable VersionNumber cassandraVersion) {
+        put(yaml, "batch_size_warn_threshold_in_kb", cassandraConfiguration.getBatchSizeFailureThresholdInKb());
```

Comment:
Even though the cassandraVersion is nullable, should we throw a log warning or something(not exactly at this line) if it's null, as ideally cassandra -v should throw the version output and our parsing has to get that value if there no unknown errors.